edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -162,8 +162,7 @@ def _verify_jwt_signature(token, jwt_issuer):
 
 def _decode_and_verify_token(token, jwt_issuer):
     options = {
-        'require_exp': True,
-        'require_iat': True,
+        'require': ["exp", "iat"],
```

Comment:
I guess this means we have insufficient test coverage. Could we add more, as part of this PR or a follow up if you want to land this fix faster?

Reply:
Added test

Reply:
https://github.com/edx/credentials/pull/1253/checks?check_run_id=3290048347#step:6:262 
This is failing in credentials. Root cause is wrong exception assertion. That's need a fix also.

Reply:
If you are building up a list of breaking change issues to search for in each repo, you should add this to the list. :)